plugin/evm/validators_default_test.go

```diff
@@ -47,13 +44,13 @@ func TestValidatorsDefault_ByEpoch(t *testing.T) {
 		t.Run(test.name, func(t *testing.T) {
 			t.Parallel()
 
-			got, err := getDefaultValidators(test.chainID)
+			got, err := NewValidatorsDefault(test.chainID)
 			test.wantErr(t, err)
 
 			if err == nil {
```

Comment:
You can skip `if err == nil {` here since `require..*` will fail before this check.